packages/example/ios/Podfile.lock

```diff
@@ -100,17 +100,18 @@ PODS:
     - GoogleUtilities/Logger
   - GoogleUtilities/UserDefaults (7.11.5):
     - GoogleUtilities/Logger
-  - Leanplum-iOS-SDK (4.0.0)
```

Comment:
is this expected to be removed?

Reply:
No, this was removed because we removed the sample app's dependency on the Leanplum Device Mode because of the conflicting symbol names between the Leanplum SDK and iOS Metrics Implementation.

Now that the conflicting symbols was fixed, added back leanplum device mode dependency to the sample app